packages/manager/modules/manager-pci-common/src/index.ts

```diff
@@ -8,8 +8,11 @@ export * from './components/region-selector';
 export * from './components/logs';
 export * from './components/modal';
 export * from './components/flavor-selector';
+export * from './components/tabs-panel';
```

Comment:
I think you are missing changes from master ;)

Reply:
i'm not sure to understand, this component has been added in this PR, what do you see missing ?

Reply:
I was referring to the export updates recently pushed to `master` (https://github.com/ovh/manager/commit/e3aaf63e420a096055c4ff2a457aa48380d93e18) but I see you are now up to date, everything's fine :)